src/components/input-field/partial-styles/trailing-icon.scss

```diff
@@ -1,11 +1,11 @@
 @use '../../../style/functions';
 
 .mdc-text-field--with-trailing-icon {
-    .mdc-text-field__icon.trailing-icon {
+    .mdc-text-field__icon--trailing {
```

Comment:
I'm not sure whether this is completely related to CSS, but type `urlAsText` and `tel` don't do anything when clicked (on the original draft PR that Adrian made). See the attached video. 


https://user-images.githubusercontent.com/35954987/125908548-43b9f449-91f2-4f1c-94af-4e4f5dac0319.mov



But when I pushed the last input-filed fixups to the draft PR, none of them works. :/ Not sure how those two fixups could affect this functionality. They seem completely irrelevant.


On my local version where I fixed a few other styling problems for readonly (not pushed yet) none of them works when clicked. See the video above.

Reply:
I pushed my local changes now.

Reply:
⚡